rskj-core/src/test/java/co/rsk/peg/LegacyErpFederationTest.java

```diff
@@ -30,10 +28,12 @@
 import java.nio.file.Paths;
 import java.time.Instant;
 import java.time.ZonedDateTime;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
+import com.google.common.collect.Lists;
```

Comment:
Is this used? In general we try to stay away from using external dependencies

Reply:
is used but can be replaced. thanks